wikiconv/conversation_reconstruction/dataflow_clean_output.py

```diff
@@ -33,7 +33,7 @@
 import traceback
```

Comment:
This code is used to replace ':' in data fields to '_' after scoring, so that the data can be imported into BigQuery. Maybe this functionality not necessary right now?

Reply:
Still necessary.